src/aics-image-viewer/components/App/index.tsx

```diff
@@ -832,6 +832,7 @@ const App: React.FC<AppProps> = (props) => {
     (): [number, number, number] => (image ? image.imageInfo.physicalPixelSize.toArray() : [1, 1, 1]),
     [image?.imageInfo.physicalPixelSize]
   );
+  const resetCamera = useMemo(() => view3d.resetCamera.bind(view3d), [view3d]);
```

Comment:
makes me wonder if there are more like this one

Reply:
This is hopefully relatively isolated, since it's pretty rare to have controls directly attached to vole-core methods. Most controls change React state, then those changes are propagated by effects, and those effects are arrow functions in which `this` behaves sensibly. This required `resetCamera` to never be wrapped in an arrow function between the button's event listener and vole-core.